test/cmd/router.sh

```diff
@@ -42,11 +42,14 @@ os::cmd::expect_failure_and_text 'oc adm router --dry-run --host-network=false -
 os::cmd::expect_failure_and_text 'oc adm router --dry-run --host-network=false --host-ports=false --router-canonical-hostname=1.2.3.4 -o yaml' 'error: canonical hostname must not be an IP address'
 # max_conn
 os::cmd::expect_success_and_text 'oc adm router --dry-run --host-network=false --host-ports=false --max-connections=14583 -o yaml' '14583'
-# ciphers 
+# ciphers
 os::cmd::expect_success_and_text 'oc adm router --dry-run --host-network=false --host-ports=false --ciphers=modern -o yaml' 'modern'
 # strict-sni
 os::cmd::expect_success_and_text 'oc adm router --dry-run --host-network=false --host-ports=false --strict-sni -o yaml' 'ROUTER_STRICT_SNI'
 
+# stats username and password
+s::cmd::expect_success_and_text 'oc adm router --dry-run --stats-password="xxx" --stats-user="statman" -o yaml' 'name: router-stats'
```

Comment:
Typo: `os::cmd`...  The leading o is missing.

This is causing your test failures.